doc/operator-guide/schema-migration-process.rst

```diff
@@ -2,8 +2,108 @@
 Schema Migration Process
 ########################
 
-* Add columns to sdm_schemas
-* Create alembic migration
-* Test migration and code to populate the new columns/tables at TTS/BTS if Summit schema is changing
+* How to add columns to sdm_schemas
+
+  - Edit the repository at https://sdm-schemas.lsst.io/ to apply your schema changes to any of the ``cdb_*.yml`` schemas.
+  - Check that valid SQL tables can be created with ``felis`` (https://felis.lsst.io/user-guide/cli.html#felis-validate) using:
+
+  .. code-block:: bash
+
+    pip install lsst-felis
+
+    felis validate [options] [edited cdb_*.yml files]
```

Comment:
Technically, this does not check that SQL tables can be created. Rather, the `validate` command checks that the schema conforms to Felis's data model.

If you want to check that a database can be created, the `felis create` command should be used.

Note that `sdm_schemas` CI will check that databases can be created for all schemas in MySQL, PostgreSQL, and SQLite, so it may not be necessary to check this locally.

Reply:
By the way, shell globs work fine with Felis commands, so this would work to validate all cdb schemas:

```
cd sdm_schemas && felis validate yml/cdb_*.yaml
```

Reply:
It makes sense to leave those details to the CI and Felis documentation then. I'm updating accordingly.